provision/tests/run-provisioners.sh

```diff
@@ -1,11 +1,15 @@
 #!/bin/bash
+DEBIAN_FRONTEND=noninteractive
+TZ=Europe/London
```

Comment:
Is UTC available as a timezone? Or is this an exact equivalent?

Reply:
This is copied from elsewhere, and just for the build of the docker image, I suspect `UTC` may work but I don't know